packages/plugin-ext/src/plugin/plugin-manager.ts

```diff
@@ -210,6 +210,7 @@ export class PluginManagerExtImpl implements PluginManagerExt, PluginManager {
 
         const deferred = new Deferred<void>();
         this.pluginActivationPromises.set(pluginId, deferred);
+        deferred.resolve();
```

Comment:
I don't think this is the right solution: if you resolve the deferred instance, you will never wait for the activation promise. But the code in "startPlugin()" seems to indicate that this is sometimes necessary (you want to wait until the plugin activation function has finished).